UnchainedLauncherCore/src/API/ServerBrowser.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Net;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System.Text.Json;
+using System.Text.Json.Serialization;
```

Comment:
Is this stuff really using both json libs?

Reply:
nope, the Newtonsoft imports are left over from when I was initially fighting with the serialization. Only the System JSON imports are used. There are tons of unused imports here, now that I look.